db/models/orders.js

```diff
@@ -40,48 +41,33 @@ const getAllOrders = async () => {
         `
     );
 
-    console.log('all orders: ', orders)
-    return await addProductsToOrders(orders)
+    return await addProductsToOrders(orders);
   } catch (error) {
     throw error;
   }
 };
 
 const getOrderById = async (id) => {
   try {
-    const {
-      rows: [order],
-    } = await client.query(
+    const { rows: arrayedOrder } = await client.query(
       `
             SELECT * FROM orders
             WHERE id = $1;
         `,
       [id]
     );
-    return await addProductsToOrders(order);
+    const [order] = await addProductsToOrders(arrayedOrder);
+    return order;
   } catch (error) {
     throw error;
   }
 };
 
-const getOrdersWithoutProducts = async () => {
-  try {
-    const { rows: orders } = await client.query(`
-      SELECT * FROM orders
-      WHERE id NOT IN (SELECT "orderId" FROM products_orders);
-    `)
-
-    return orders;
-  } catch(error) {
-    throw error; 
-  }
-}
-
-const getOrdersByUser = async ({ username }) => {
+const getOrdersByUser = async (username) => {
   try {
     const { rows: orders } = await client.query(
       `
-            SELECT orders.*, users.username AS "customerName"
+            SELECT orders.*, users.username, users.id
```

Comment:
the users id will take over the property of id
so if i have an object like this  
{
  id: userId => orderId
}